src/platform/site-wide/side-nav/components/NavItemRow.js

```diff
@@ -47,6 +45,9 @@ const NavItemRow = ({ depth, item, toggleItemExpanded }) => {
   if (hasChildren) {
     return (
       <a
+        {...!isExpanded &&
```

Comment:
Did we mean to use a spread operator here?

Reply:
Good q, @kelsonic. Looks like an inadvertent addition, but @brangi made this commit, so I’m gonna defer to him.

Reply:
@kelsonic  @ethanteague pushed a fix